### PR TITLE
Change billing archiving command to run for given months

### PR DESCRIPTION
**Pull request by justinvdm**

### CODE REVIEW

**Comment by justinvdm** [comment]
Ready for review.

File: go/billing/tests/test_go_archive_transactions.py [hunk 1]
```diff
@@ -36,41 +36,64 @@ def assert_remaining_transactions(self, transactions):
...
+        [dec_archive] = TransactionArchive.objects.filter(
+            account=self.account,
+            from_date=datetime(2014, 12, 1),
+            to_date=datetime(2014, 12, 31),
+            status=TransactionArchive.STATUS_TRANSACTIONS_UPLOADED)
 
         s3_bucket = bucket.get_s3_bucket()
-        [s3_key] = list(s3_bucket.list())
-        self.assertEqual(s3_key.key, archive.filename)
+        [nov_s3_key, dec_s3_key] = list(s3_bucket.list())
```
Comment (hodgestar) [inline go/billing/tests/test_go_archive_transactions.py]: I wonder if these are guaranteed to be in the correct order or whether we should explicitly sort the keys returned by `s3_bucket.list()`?

File: go/billing/tests/test_go_archive_transactions.py [hunk 2]
```diff
@@ -78,33 +101,45 @@ def test_archive_transactions_with_deletion(self):
...
+        [dec_archive] = TransactionArchive.objects.filter(
+            account=self.account,
+            from_date=datetime(2014, 12, 1),
+            to_date=datetime(2014, 12, 31),
+            status=TransactionArchive.STATUS_ARCHIVE_COMPLETED)
 
         s3_bucket = bucket.get_s3_bucket()
-        [s3_key] = list(s3_bucket.list())
-        self.assertEqual(s3_key.key, archive.filename)
+        [nov_s3_key, dec_s3_key] = list(s3_bucket.list())
```
Comment (hodgestar) [inline go/billing/tests/test_go_archive_transactions.py]: Same questions here.

**Comment by hodgestar** [comment]
Otherwise looks good.

**Comment by hodgestar** [comment]
:+1:

**Pull request merged**